gocd-server/tests/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+TESTDIR="$(dirname "${0}")"
+PLANDIR="$(dirname "${TESTDIR}")"
+SKIPBUILD=${SKIPBUILD:-0}
+
+hab pkg install --binlink core/bats
+
+source "${PLANDIR}/plan.sh"
+
+if [ "${SKIPBUILD}" -eq 0 ]; then
+  set -e
+  pushd "${PLANDIR}" > /dev/null
+  build
+  source results/last_build.env
+  hab pkg install --binlink --force "results/${pkg_artifact}"
+  hab pkg install --binlink --force core/curl
```

Comment:
Please move the `core/curl` installation outside the `SKIPBUILD` check, and remove the `--force` option. This will ensure that its installed before any tests run, no matter the `SKIPBUILD` setting.